CMakeLists.txt

```diff
@@ -13,9 +13,20 @@ project(git
 		VERSION ${git_version}
 		LANGUAGES C)
 
+#[[
+NOTE:
+FOR LINUX USE THIS GIT REPO https://github.com/git/git
+FOR WINDOWS USE THIS GIT REPO https://github.com/git-for-windows/git
+]]
+
 #TODO gitk git-gui gitweb
+#TODO Enable NLS on windows natively
 #TODO Add pcre support
 
+#NOTE
+#On windows when building with shared libraries
+#don't forget to append their location to the PATH variable.
```

Comment:
Do we still need to take care of this? If this is not actually actionable, we should drop that comment altogether.